identity-account/src/identity/identity_state.rs

```diff
@@ -45,7 +49,39 @@ use crate::types::KeyLocation;
 type Properties = VerifiableProperties<BaseProperties>;
 type BaseDocument = CoreDocument<Properties, Object, Object>;
 
-pub type RemoteEd25519<'a, T> = JcsEd25519<RemoteSign<'a, T>>;
+pub struct RemoteEd25519;
```

Comment:
Nit: can/should we move the definition and implementation of `RemoteEd25519` from `identity_state.rs` to `remote.rs`?